server/sonar-web/src/main/js/apps/quality-gates/components/Projects.js

```diff
@@ -18,44 +18,55 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import React from 'react';
-import ProjectsView from '../views/gate-projects-view';
+import escapeHtml from 'escape-html';
+import SelectList from '../../../components/SelectList';
+import { translate } from '../../../helpers/l10n';
+import { getBaseUrl } from '../../../helpers/urls';
 
 export default class Projects extends React.PureComponent {
   componentDidMount() {
-    this.renderView();
+    this.renderSelectList();
   }
 
-  componentWillUpdate() {
-    this.destroyView();
-  }
-
-  componentDidUpdate() {
-    this.renderView();
-  }
+  renderSelectList = () => {
+    if (!this.container) return;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add curly braces around the nested statement(s) in this "if" block. ([javascript:CurlyBraces](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ACurlyBraces))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fless-backbone-more-react&issues=AWG-K8_X_Zsgh39atBHm&open=AWG-K8_X_Zsgh39atBHm)

[ISSUE_KEY]: AWG-K8_X_Zsgh39atBHm